pkg/restapi/gnap/operations.go

```diff
@@ -25,7 +25,8 @@ import (
 var logger = log.New("auth-restapi") //nolint:gochecknoglobals
 
 const (
-	gnapBasePath = "/gnap"
+	hydraLoginPath = "/hydra/login"
```

Comment:
we don't need hydra prefix here - just use `loginPath = "/login"`

In case of OIDC, hydra used to act as OIDC provider for auth project and used to call this API. In case of GNAP, client directly calls it.